rosidl_generator_dds_idl/resource/action.idl.em

```diff
@@ -5,43 +5,49 @@ TEMPLATE(
     package_name=package_name,
     interface_path=interface_path,
     message=action.goal,
-    get_post_struct_lines=get_post_struct_lines
+    get_post_struct_lines=get_post_struct_lines,
+    idl_typename=idl_typename, idl_literal=idl_literal,
```

Comment:
style nit: I would expect each argument on a separate line.

Reply:
> I would expect each argument on a separate line.

I don't think we have that in our style guide and our code base certainly doesn't do it either. So I will keep it as is since it keeps each invocation more condense.